## 82% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 5
Expected to fail tests: 4
Passed tests: 42
```
```
IDs of failed tests: 727230,702809,703503,727229,702947 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/status-bar-color-ios&PR_ID=20540&APK_NAME=20540.apk&TR_CASE_IDS=727230,702809,703503,727229,702947">Rerun failed tests</a></li><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588190">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:152: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588189">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:132: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['wallet_address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588172">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div># STEP: Checking unread indicators</div></blockquote></p>

```
activity_center/test_activity_center.py:312: in test_activity_center_reply_read_unread_delete_filter_swipe
    self.home_1.notifications_unread_badge.wait_for_visibility_of_element(120)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: BaseElement by accessibility id:`activity-center-unread-count` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9237dffe3e794c669bd9b9b38d26fb74?auth=c4ea39b68f3ee67e113801cefd74db92#132">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff40afbc07b64657848591d24ce3f809?auth=1b070b96eee0066329dde309b701e043#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588185">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `communities-home-discover-card` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:261: in test_community_discovery
    self.home.community_card_item.wait_for_elements(seconds=120)
../views/base_element.py:138: in wait_for_elements
    raise TimeoutException(
 Device 1:  BaseElement by accessibility id:`community-card-item` is not found on the screen after wait_for_elements
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588156">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 bold text in asterics is not displayed with markdown in community channel for the recipient (device 1) 
E   
E    bold text in underscores is not displayed with markdown in community channel for the recipient (device 1)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7724a915c8eb4c5286605b579a593158?auth=ca0357ee95eccca1bc921deac9bbe213#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9895e83d4ee64cbab0114dfb376fc583?auth=6329fad815dd3b6b7691546ed327b04a#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588173">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7724a915c8eb4c5286605b579a593158?auth=ca0357ee95eccca1bc921deac9bbe213#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9895e83d4ee64cbab0114dfb376fc583?auth=6329fad815dd3b6b7691546ed327b04a#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588186">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7724a915c8eb4c5286605b579a593158?auth=ca0357ee95eccca1bc921deac9bbe213#498">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9895e83d4ee64cbab0114dfb376fc583?auth=6329fad815dd3b6b7691546ed327b04a#475">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588183">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#475">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#273">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#242">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588192">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:256: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4e3c4d87d134eaa890499eb67d9b490?auth=c2a946bf4dc3c58112cda7e8ab39996a#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (42)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588144">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588180">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#141">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588179">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588154">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588155">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7468d7a23840b689145079ccd9bf30?auth=a262134875077e2870a02386ae101a5a#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a86fe57f763a4ed4ba577672f05fc319?auth=b9081483ba18282062fa318d387aaef3#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fae0ccc305e5454a8d66b371726d2f8c?auth=b4efea34d5c30df9fb0a42a1f50832c9#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588153">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7724a915c8eb4c5286605b579a593158?auth=ca0357ee95eccca1bc921deac9bbe213#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9895e83d4ee64cbab0114dfb376fc583?auth=6329fad815dd3b6b7691546ed327b04a#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588164">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7724a915c8eb4c5286605b579a593158?auth=ca0357ee95eccca1bc921deac9bbe213#298">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9895e83d4ee64cbab0114dfb376fc583?auth=6329fad815dd3b6b7691546ed327b04a#269">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588187">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c7a0101282a144d69e091f8e6b2bda03?auth=f3d98f439dd7a80f54a180355b81b85d#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588148">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c7a0101282a144d69e091f8e6b2bda03?auth=f3d98f439dd7a80f54a180355b81b85d#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588149">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7fd10e5ddbe946cb82eb469e92340792?auth=cd9462ffdfee678ea3a9890e24947488#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/763d89994f404bbaa0b66c0df87b1599?auth=1040f40ecd9ce176d2bf71561d1b55d1#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588167">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7fd10e5ddbe946cb82eb469e92340792?auth=cd9462ffdfee678ea3a9890e24947488#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/763d89994f404bbaa0b66c0df87b1599?auth=1040f40ecd9ce176d2bf71561d1b55d1#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588166">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7fd10e5ddbe946cb82eb469e92340792?auth=cd9462ffdfee678ea3a9890e24947488#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/763d89994f404bbaa0b66c0df87b1599?auth=1040f40ecd9ce176d2bf71561d1b55d1#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588150">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#147">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588145">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#554">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#400">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588157">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#413">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588147">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#209">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588168">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#462">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#290">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588182">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#340">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588143">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#209">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588142">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ac2b26d74864e6291096634abe9e813?auth=9ee5368725e9745c0fd294caf148b9f2#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d2bc04897a0e4361acb27d2516ce50b7?auth=32fcae295d8f3fc1314ae44f43532ca2#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588178">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#233">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588169">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#225">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588160">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#285">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588176">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#552">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#469">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588171">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#425">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#393">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588188">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#587">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#495">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588159">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588158">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588163">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#330">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588162">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588161">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0fa31193f7cf4102862b907315e8f00c?auth=2eb066d1fdf0c6586a35f8f9c29fd38d#395">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a5f400b20f346e495db9c6c3af3b48b?auth=7858707d39a431a5eb8811db2b98c4ff#363">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588174">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84cf2a549a0045b6be4e195dfb4ae9e0?auth=756d6d05951e038d8ef2a96d8259911b#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1157fc68775643c984c6eae2b6a9d02a?auth=613d5a569720e63c34cd5f8a63c584fc#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588175">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84cf2a549a0045b6be4e195dfb4ae9e0?auth=756d6d05951e038d8ef2a96d8259911b#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1157fc68775643c984c6eae2b6a9d02a?auth=613d5a569720e63c34cd5f8a63c584fc#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588152">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c8eef57e1d449dda4c7c7a9652d33be?auth=fff0a6959850fe42298cadfe88f5cded#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c00dfd33be2541b687291cf5273490e4?auth=bf2eff13a59539fb58bbe460a37cc1d2#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588151">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c8eef57e1d449dda4c7c7a9652d33be?auth=fff0a6959850fe42298cadfe88f5cded#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c00dfd33be2541b687291cf5273490e4?auth=bf2eff13a59539fb58bbe460a37cc1d2#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588184">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c8eef57e1d449dda4c7c7a9652d33be?auth=fff0a6959850fe42298cadfe88f5cded#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c00dfd33be2541b687291cf5273490e4?auth=bf2eff13a59539fb58bbe460a37cc1d2#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588177">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588146">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588170">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588165">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588181">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2242201a8f3841b4a68224095a276573?auth=cea250cd9fd3ca56c9c87aa1cf28d4b6#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3588191">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a4e3c4d87d134eaa890499eb67d9b490?auth=c2a946bf4dc3c58112cda7e8ab39996a#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/ios/StatusIm/Info.plist
+++ b/ios/StatusIm/Info.plist
@@ -119,8 +119,6 @@
 	<array>
 		<string>armv7</string>
 	</array>
-	<key>UIStatusBarStyle</key>
-	<string>UIStatusBarStyleLightContent</string>
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
@@ -133,7 +131,7 @@
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<false/>
+	<true/>
 	<key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
 	<array>
 		<string>A00000080400010101</string>

--- a/src/legacy/status_im/multiaccounts/logout/core.cljs
+++ b/src/legacy/status_im/multiaccounts/logout/core.cljs
@@ -28,7 +28,7 @@
   [{:keys [db] :as cofx} {:keys [auth-method logout?]}]
   (let [key-uid (get-in db [:profile/profile :key-uid])]
     (rf/merge cofx
-              {:set-root                               :progress
+              {:dispatch                               [:init-root :progress]
                :effects.shell/reset-state              nil
                :hide-popover                           nil
                ::logout                                nil

--- a/src/legacy/status_im/ui/screens/screens.cljs
+++ b/src/legacy/status_im/ui/screens/screens.cljs
@@ -53,7 +53,8 @@
   []
   [;;PROGRESS
    {:name      :progress
-    :options   {:insets {:top? true}}
+    :options   {:insets {:top? true}
+                :theme  :dark}
     :component progress/progress}
 
    ;;COMMUNITY

--- a/src/status_im/contexts/onboarding/events.cljs
+++ b/src/status_im/contexts/onboarding/events.cljs
@@ -89,7 +89,7 @@
     (merge
      {:db (assoc db :profile/profiles-overview multiaccounts)}
      (when-not (seq multiaccounts)
-       {:set-root :screen/onboarding.intro}))))
+       {:dispatch [:init-root :screen/onboarding.intro]}))))
 
 (rf/defn password-set
   {:events [:onboarding/password-set]}

--- a/src/status_im/contexts/profile/events.cljs
+++ b/src/status_im/contexts/profile/events.cljs
@@ -45,14 +45,14 @@
                   (assoc :profile/profiles-overview profiles)
                   (update :profile/login #(select-profile % key-uid)))
               db)
-        :fx [[:set-root :screen/profile.profiles]
+        :fx [[:dispatch [:init-root :screen/profile.profiles]]
              (when key-uid
                [:effects.biometric/check-if-available
                 {:key-uid    key-uid
                  :on-success (fn [auth-method]
                                (rf/dispatch [:profile.login/check-biometric-success key-uid
                                              auth-method]))}])]})
-     {:fx [[:set-root :screen/onboarding.intro]]})))
+     {:fx [[:dispatch [:init-root :screen/onboarding.intro]]]})))
 
 (rf/reg-event-fx
  :profile/update-setting-from-backup

--- a/src/status_im/contexts/profile/login/events.cljs
+++ b/src/status_im/contexts/profile/login/events.cljs
@@ -74,7 +74,7 @@
 
                 (cond
                   pairing-completed?
-                  [[:set-root :screen/onboarding.syncing-results]]
+                  [[:dispatch [:init-root :screen/onboarding.syncing-results]]]
 
                   (get db :onboarding/new-account?)
                   [[:dispatch [:onboarding/finalize-setup]]]
@@ -85,7 +85,7 @@
                          constants/theme-type-dark)
                      :shell-stack
                      false]]
-                   [:set-root :shell-stack]
+                   [:dispatch [:init-root :shell-stack]]
                    [:dispatch [:profile/show-testnet-mode-banner-if-enabled]]]))})))
 
 ;; login phase 2: we want to load and show chats faster, so we split login into 2 phases
@@ -179,7 +179,7 @@
      {:db (-> db
               (assoc-in [:profile/login :password] password)
               (assoc-in [:profile/login :processing] true))
-      :fx [[:set-root :progress]
+      :fx [[:dispatch [:init-root :progress]]
            [:effects.profile/login
             [(get-in db [:profile/login :key-uid])
              (security/safe-unmask-data password)]]]})))

--- a/src/status_im/navigation/effects.cljs
+++ b/src/status_im/navigation/effects.cljs
@@ -2,7 +2,6 @@
   (:require
     [quo.foundations.colors :as colors]
     [quo.theme]
-    [react-native.core :as rn]
     [react-native.navigation :as navigation]
     [react-native.platform :as platform]
     [react-native.share :as share]
@@ -17,37 +16,36 @@
 
 (defn get-status-nav-color
   [view-id theme]
-  (let [theme (or (get-in views/screens [view-id :options :theme])
-                  theme)
-        [rnn-status-bar rn-status-bar]
-        (if (or (= theme :dark)
-                @state/alert-banner-shown?
-                (and (= view-id :shell-stack) (not (jump-to.utils/home-stack-open?))))
-          [:light "light-content"]
-          [:dark "dark-content"])
-        home-stack? (some #(= view-id %) shell.constants/stacks-ids)
+  (let [theme            (or (get-in views/screens [view-id :options :theme])
+                             theme)
+        status-bar-theme (if (or (= theme :dark)
+                                 @state/alert-banner-shown?
+                                 (and (= view-id :shell-stack)
+                                      (not (jump-to.utils/home-stack-open?))))
+                           :light
+                           :dark)
+        home-stack?      (some #(= view-id %) shell.constants/stacks-ids)
         ;; Home screen nav bar always dark due to bottom tabs
-        nav-bar-color (if (or home-stack?
-                              (= view-id :shell-stack)
-                              (= theme :dark))
-                        colors/neutral-100
-                        colors/white)
-        comp-id (if (or home-stack?
-                        (jump-to.utils/shell-navigation? view-id)
-                        (= view-id :shell))
-                  :shell-stack
-                  view-id)]
-    [rnn-status-bar rn-status-bar nav-bar-color comp-id]))
+        nav-bar-color    (if (or home-stack?
+                                 (= view-id :shell-stack)
+                                 (= theme :dark))
+                           colors/neutral-100
+                           colors/white)
+        comp-id          (if (or home-stack?
+                                 (jump-to.utils/shell-navigation? view-id)
+                                 (= view-id :shell))
+                           :shell-stack
+                           view-id)]
+    [status-bar-theme nav-bar-color comp-id]))
 
 (defn reload-status-nav-color-fx
   [[view-id theme]]
   (when (and (= @state/root-id :shell-stack) view-id)
-    (let [[rnn-status-bar rn-status-bar nav-bar-color comp-id] (get-status-nav-color view-id theme)]
-      (if platform/ios?
-        (rn/set-status-bar-style rn-status-bar true)
+    (let [[status-bar-theme nav-bar-color comp-id] (get-status-nav-color view-id theme)]
+      (when platform/android?
         (navigation/merge-options
          (name comp-id)
-         {:statusBar     {:style rnn-status-bar}
+         {:statusBar     {:style status-bar-theme}
           :navigationBar {:backgroundColor nav-bar-color}})))))
 
 (rf/reg-fx :reload-status-nav-color-fx reload-status-nav-color-fx)
@@ -72,8 +70,9 @@
 ;;;; Root
 
 (rf/reg-fx :set-root
- (fn [root-id]
-   (let [root (get (roots/roots) root-id)]
+ (fn [[root-id theme]]
+   (let [[status-bar-theme] (get-status-nav-color root-id theme)
+         root               (get (roots/roots status-bar-theme) root-id)]
      (dismiss-all-modals)
      (rf/dispatch [:profile.settings/switch-theme
                    (get roots/themes root-id)
@@ -212,9 +211,9 @@
 
 (rf/reg-fx :show-toasts
  (fn [[view-id theme]]
-   (let [[rnn-status-bar nav-bar-color] (get-status-nav-color view-id theme)]
+   (let [[status-bar-theme nav-bar-color] (get-status-nav-color view-id theme)]
      (show-overlay "toasts"
-                   (assoc (options/statusbar-and-navbar-options nil rnn-status-bar nav-bar-color)
+                   (assoc (options/statusbar-and-navbar-options nil status-bar-theme nav-bar-color)
                           :overlay
                           {:interceptTouchOutside false})))))
 
@@ -233,7 +232,10 @@
 ;;;; Alert Banner
 (rf/reg-fx :show-alert-banner
  (fn [[view-id theme]]
-   (show-overlay "alert-banner" {:overlay {:interceptTouchOutside false}})
+   (show-overlay "alert-banner"
+                 (assoc (options/statusbar-and-navbar-options nil :light nil)
+                        :overlay
+                        {:interceptTouchOutside false}))
    (reset! state/alert-banner-shown? true)
    (reload-status-nav-color-fx [view-id theme])))
 

--- a/src/status_im/navigation/events.cljs
+++ b/src/status_im/navigation/events.cljs
@@ -71,8 +71,8 @@
 
 (rf/defn init-root
   {:events [:init-root]}
-  [_ root-id]
-  {:set-root root-id})
+  [{:keys [db]} root-id]
+  {:set-root [root-id (:theme db)]})
 
 (rf/defn set-stack-root
   {:events [:set-stack-root]}

--- a/src/status_im/navigation/roots.cljs
+++ b/src/status_im/navigation/roots.cljs
@@ -15,7 +15,7 @@
    :shell-stack             nil})
 
 (defn roots-internal
-  []
+  [status-bar-theme]
   {:screen/onboarding.intro
    {:root
     {:stack {:id       :screen/onboarding.intro
@@ -28,7 +28,8 @@
              :children [{:component {:name    :shell-stack
                                      :id      :shell-stack
                                      :options (options/root-options
-                                               {:nav-bar-color colors/neutral-100})}}]}}}
+                                               {:nav-bar-color    colors/neutral-100
+                                                :status-bar-theme status-bar-theme})}}]}}}
    :screen/profile.profiles
    {:root
     {:stack {:id       :screen/profile.profiles
@@ -51,17 +52,18 @@
                                            :options (options/dark-root-options)}}]}}}})
 
 (defn old-roots
-  []
+  [status-bar-theme]
   {:progress
    {:root {:stack {:children [{:component {:name    :progress
                                            :id      :progress
-                                           :options (options/root-options nil)}}]
+                                           :options (options/root-options {:status-bar-theme
+                                                                           status-bar-theme})}}]
                    :options  (assoc (options/root-options nil)
                                     :topBar
                                     {:visible false})}}}})
 
 (defn roots
-  []
+  [status-bar-theme]
   (merge
-   (old-roots)
-   (roots-internal)))
+   (old-roots status-bar-theme)
+   (roots-internal status-bar-theme)))

